SPECS/mariner-release/mariner-release.spec

```diff
@@ -62,6 +62,9 @@ EOF
 %config(noreplace) %{_sysconfdir}/issue.net
 
 %changelog
+* Wed Jul 27 2022 Minghe Ren <mingheren@microsoft.com> - 2.0-16
+- remove issue.net kernel part as it is not showing properly by openssh
```

Comment:
Could you add a longer comment explaining that sshd doesn't support the old-style telnet escape sequences, so they were removed?